cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -86,12 +91,54 @@ func (m *Manager) Deploy(ctx context.Context) (*DeployResult, error) {
 		return nil, fmt.Errorf("updating environment with deployment outputs: %w", err)
 	}
 
+	stAccount, hasBindMountOperation := m.env.LookupEnv(operationsBindMount)
+	if hasBindMountOperation {
+		for key, value := range m.env.Dotenv() {
+			if key != operationsBindMount && strings.HasPrefix(key, operationsBindMountPrefix) {
+				valueParts := strings.Split(value, ",")
+				if len(valueParts) != 2 {
+					return nil, fmt.Errorf("invalid bind mount operation value: %s", value)
+				}
+				fileShareName, source := valueParts[0], valueParts[1]
+				if err := bindMountOperation(
+					ctx, m.fileShareService, m.env.GetSubscriptionId(), stAccount, fileShareName, source); err != nil {
```

Comment:
Is there a different way to pass the file paths to upload not via env? That seems slightly strange. The interpretation of the local paths are also tied to a given machine, I'm not sure how we're thinking about the paths here to work on CI vs. local. 

I do like the env value for the nominal remote storage account to upload to, just not the paths.

Reply:
The path that is bind is usually within the repo, so it should work in CI as well.